### ISSUE TITLE
Could the author provide more information on the implementation settings of M-ADA on Cifar10-C ?

### ISSUE THREAD

**Issue by txsing** [joffery/M-ADA#11]
- The initial learning rate is 0.1 with a linear decay and the number of epochs is 200. What is the decay rate?

- What are the values of hyper parameters  `K`, `T_adv`, `T_min`, `gamma`, `beta`, `lr_max` etc.?

- What are the transformers (Crop \ Normalzaiton etc) used in the training phase? 

I tried several times following the settings of origin WRN paper use M-ADA code base, but failed to get the expected results/
